src/js/utils/index.js

```diff
@@ -0,0 +1,20 @@
+import {
+  mainElementController,
+  footerElementController,
+  taskElementController,
+  newTodoInputElementController,
+  todoElementController,
+} from "./dom-manipulation";
+import { existTasksInTodoList } from "./utils";
+
+export const DOM = {
+  controllers: {
+    main: mainElementController,
```

Comment:
It should make more sense to call all controllers to elements, for example:
- main to mainEl
- footer to footerEl
- taskItem to taskItemEl
- newTodoInput to newTodoInputEl
- todoList to todoListEl